react-native-sdk/index.js

```diff
@@ -0,0 +1,89 @@
+import 'react-native-gesture-handler';
+
+// Apply all necessary polyfills as early as possible to make sure anything imported henceforth
+// sees them.
+import 'react-native-get-random-values';
+import './react/features/mobile/polyfills';
+// eslint-disable-next-line no-unused-vars
+import React, { useState, useEffect, useImperativeHandle, forwardRef, useRef } from 'react';
+// eslint-disable-next-line no-unused-vars
+import { View } from 'react-native';
+
+
+// eslint-disable-next-line no-unused-vars
+import { App } from './react/features/app/components/App.native';
+// eslint-disable-next-line no-unused-vars
+import JitsiThemePaperProvider from './react/features/base/ui/components/JitsiThemeProvider';
+import { closeJitsiMeeting } from './react/features/mobile/react-native-sdk/functions';
+import { convertPropsToURL } from './utils';
+
+
+type AppProps = {
+    flags: [];
+    meetingOptions: {
+        domain: string;
+        roomName: string;
+        onReadyToClose?: Function;
+        onConferenceJoined?: Function;
+        onConferenceWillJoin?: Function;
+        onConferenceLeft?: Function;
+        settings?: {
+            startWithAudioMuted?: boolean;
+            startAudioOnly?: boolean;
+            startWithVideoMuted?: boolean;
+            startWithReactionsMuted?: boolean;
+        }
+    }
+};
+
+/**
+ * Main React Native SDK component that displays a Jitsi Meet conference and gets all required params as props
+ */
+const JitsiMeetView = forwardRef((props: AppProps, ref) => {
```

Comment:
Instead of calling it this way, let's try to align it a bit with the react SDK and call it JitsiMeeting.